kernel/path/Cargo.toml

```diff
@@ -3,26 +3,8 @@ name = "path"
 version = "0.1.0"
 authors = ["Andrew Pham <apham727@gmail.com>, Christine Wang <chrissywang54@gmail.com"]
```

Comment:
with this significant of a restructuring, you can add yourself as primary author.